cmd/oci-runtime-tool/validate.go

```diff
@@ -465,17 +450,33 @@ func envValid(env string) bool {
 	return true
 }
 
-func capValid(capability string) bool {
-	for _, val := range defaultCaps {
-		if val == capability {
-			return true
-		}
-	}
+func capValid(c string, hostSpecific bool) bool {
+    cp := strings.ToUpper(c)
+
+    for _, cap := range capability.List() {
+        if cp == fmt.Sprintf("CAP_%s", strings.ToUpper(cap.String())) {
+            if hostSpecific && cap > lastCap() {
+				return false
+            }
+            return true
+        }
+    }
+
 	return false
 }
 
+func lastCap() capability.Cap {
```

Comment:
I don't like duplicating code between the generate and validate libraries.  Validation seems more fundamental to me, so can we make a public `LastCap` function here and import that into the generate library?